### ISSUE TITLE
Incorrect Language Tracking for Mojo Programming Language

### ISSUE THREAD

**Issue by Tgentil** [wakatime/vscode-wakatime#365]
### Description

I have noticed that while coding using the new `Mojo` programming language in VS Code with the Wakatime extension enabled, the time I spent coding is being wrongly attributed to ActionScript instead of Mojo. I suspect this might be due to the fact that I used `.🔥` file extension instead of `.mojo`.

### Steps to Reproduce

1. Set up the environment following the guide available at [Modular Developer](https://developer.modular.com/download).
2. Install in VS Code, the WSL extension, and the Mojo extension.
3. Install Ubuntu 22.04 for WSL and open it.
4. Install the Modular CLI and Mojo SDK using the commands:
```bash
curl https://get.modular.com | \
  MODULAR_AUTH=mut_cc41c6b08c624fe397cdb5904a7e9a02 \
  sh -
  ```
5. Open the Ubuntu workspace in VS Code with the command: `code .`.
6. Connect your Wakatime account.
7. Start coding in Mojo using `.🔥` file extension.

### Expected Behavior

The time spent coding should be attributed to the Mojo programming language in Wakatime statistics.

### Actual Behavior

The coding time is being attributed to ActionScript.

### Possible Solution

Ensure that the Wakatime extension properly recognizes the `.🔥` and `.mojo` file extensions as associated with the Mojo programming language, not ActionScript.

Thank you for your attention, and I look forward to a resolution.

**Comment by gandarez** [comment]
@Tgentil what's the file extensions represented by the fire icon?

**Comment by Tgentil** [comment]
Hello @gandarez , 

The flame icon is used to represent the `.mojo` file extension, as outlined in the [official Mojo documentation](https://docs.modular.com/mojo/manual/get-started/hello-world.html#build-and-run-mojo-source-files). According to the documentation, Mojo source files can be identified with either the `.mojo` or the `.🔥` file extension. Here's an excerpt from the documentation for your reference:

> "Now let’s print “Hello, world” with a source file. Mojo source files are identified with either the .mojo or .🔥 file extension.

To give you a bit more context on the Mojo programming language, I'd recommend checking out this short [video by Fireship on YouTube](https://www.youtube.com/watch?v=V4gGJ7XXlC0&ab_channel=Fireship) that provides a nice overview of the language.

I hope this helps to clarify. Please let me know if there is anything else you'd like to know.

**Comment by gandarez** [comment]
@Tgentil wakatime-cli has been released v1.80.0
